Automattic/wp-calypso
# Change the live preview preparing copy

Related to https://github.com/Automattic/wp-calypso/pull/80353

## Proposed Changes

This PR changes the "live preview preparing" copy, as @autumnfjeld suggested in p1696529762314299/1696498023.863089-slack-CRWCHQGUB. 

Let's use the static and simple copy for now https://github.com/Automattic/wp-calypso/pull/82696#issuecomment-1750049397.

## Testing Instructions



* Go to Appearance > Themes
* Click the three-dots option on the Theme Card
* Click the `Preview & Customize` option (that should show up on a free theme)
* Verify the copy is changed to `Preparing theme preview` when loading

<img width="831" alt="Screenshot 2023-10-06 at 16 05 30" src="https://github.com/Automattic/wp-calypso/assets/5287479/333946f8-a2c9-4d10-8041-e5fe9be72c0b">

## Pre-merge Checklist
